cartography/intel/aws/ec2/launch_templates.py

```diff
@@ -55,20 +67,24 @@ def load_launch_templates(
 
 @timeit
 @aws_handle_regions
-def get_launch_template_versions(
+def get_launch_template_versions_by_template(
         boto3_session: boto3.session.Session,
-        templates: list[dict[str, Any]],
+        template: dict[str, Any],
```

Comment:
non-block: consider passing in the template_id as a str to this function since I don't think the whole object is necessary.